package-lock.json

```diff
@@ -1,8 +1,3029 @@
 {
   "name": "eslint-plugin-tailwindcss",
   "version": "1.10.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Looks like you are using a newer version of node.

As of today the LTS version of node is:

> Latest LTS Version: 14.17.0 (includes npm 6.14.13)

While you might be using:

> Latest Current Version: 16.3.0 (includes npm 7.15.1)

About the [`lockfileVersion`](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#lockfileversion)

- `1`: The lockfile version used by `npm v5` and `v6`.
- `2`: The lockfile version used by `npm v7`, which is backwards compatible to `v1` lockfiles.

I think we should stick to `v1` for now.

Reply:
Will do it.